includes/views/advanced-status.php

```diff
@@ -95,15 +101,15 @@
 		)
 	);
 }
-if ( ( $gm = extension_loaded( 'gmagick' ) ) || ( $im = extension_loaded( 'imagick' ) ) ) {
-	$server_configs['GMagick or IMagick']['value'] = ( $im ? 'IMagick '.phpversion( 'imagick' ) : 'GMagick '.phpversion( 'gmagick' ) );
+
+if ( extension_loaded( 'gmagick' ) || ( $im = extension_loaded( 'imagick' ) ) ) {
+	$server_configs['GMagick or IMagick']['value'] = ( $im ? 'IMagick ' . phpversion( 'imagick' ) : 'GMagick ' . phpversion( 'gmagick' ) );
```

Comment:
I think if the `extension_loaded( 'gmagick' )` returns `true` in the condition, then `$im` will be undefined.